bigquery/nox.py

```diff
@@ -36,7 +36,7 @@ def default(session):
     """
     # Install all test dependencies, then install this package in-place.
     session.install('mock', 'pytest', 'pytest-cov', *LOCAL_DEPS)
-    session.install('-e', '.')
+    session.install('-e', '.[pandas]')
```

Comment:
I'd be happier if there was at least one session that ran the unit tests without Pandas.

Maybe @dhermes @lukesneeringer @jonparrott are more comfortable with the risk that we accidentally convert the optional dependency into a required dependency at some point?

Looks like the snippets session doesn't include Pandas, but maybe that's more accidental than intentional?

Reply:
I'm not worried about converting the optional dependency, but I'm also fine with running a session without pandas.